src/main/groovy/net/minecrell/gradle/licenser/tasks/LicenseCheck.groovy

```diff
@@ -26,12 +26,21 @@ package net.minecrell.gradle.licenser.tasks
 
 import net.minecrell.gradle.licenser.LicenseViolationException
 import org.gradle.api.file.FileVisitDetails
+import org.gradle.api.tasks.Input
 import org.gradle.api.tasks.TaskAction
 import org.gradle.api.tasks.VerificationTask
 
-class LicenseCheck extends LicenseTask implements VerificationTask {
+class LicenseCheck extends LicenseTask implements VerificationTask
```

Comment:
Looks like accidental newline?

Reply:
It might've been caused when I did a reformat of the entire file (as evident on line https://github.com/Minecrell/licenser/pull/12/files#diff-bb87b6dea41369987c29ee12dd8142a7R53)